maestro-cli/src/main/java/maestro/cli/device/DeviceService.kt

```diff
@@ -122,19 +115,7 @@ object DeviceService {
                 process
                     .waitFor(1000, TimeUnit.MILLISECONDS)
                 process.exitValue() == 0
-            } || error("Simulator failed to boot")
-
-            // Test if idb can get accessibility info elements with non-zero frame with
-            logger.warning("Waiting for Accessibility info to become available..")
-            MaestroTimer.retryUntilTrue(timeoutMs = 20000, delayMs = 100) {
```

Comment:
We still need to wait for IDB to boot, otherwise there is a risk that `tap` command will fail because IDB is not ready. We can't use `contentDescriptor` for that anymore, so we need some other way to deduce that

Reply:
True that I have used tap result to check if idb is ready. Dispatching tap event on 0,0 coordinates and see if it's Ok.